kvmagent/kvmagent/test/unittest_tools/unittest_pypi_source/pypi_source_builder.sh

```
@@ -0,0 +1,151 @@
+        fi
+    done
+
+    if [[ x"$output" = x"version" ]]; then  # output=version
+        echo "${parts[$split_at]}"
+    else  # output=package-name
+        if [[ $split_at -ge len ]]; then
+            echo "$clean" | tr '[:upper:]' '[:lower:]'
+            return
+        fi
+        local name_parts=("${parts[@]:0:split_at}")
+        local name=$(IFS=-; echo "${name_parts[*]}")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)